### PR TITLE
[GOG]: Get game builds only from main branch

### PR DESCRIPTION
**Pull request by imLinguin**
This PR addresses https://github.com/Heroic-Games-Launcher/heroic-gogdl/issues/34  
Here we are making sure we use builds from main branch only. This is temporal change and branching will be further customizable in slowly progressing gogdl refactor. 

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by flavioislima — APPROVED** [review]
LGTM
Not sure how to test this though.

**Comment by imLinguin** [comment]
Stardew Valley is great example, builds from compat branch were picked before this change. Other game like that is the one mentioned in the gogdl issue. If you own those you can test them

File: src/backend/storeManagers/gog/library.ts [hunk 1]
```diff
@@ -676,7 +677,9 @@ export async function getMetaResponse(
         'If-None-Match': etag
       }
     : undefined
-  const metaUrl = buildData.data?.items[0]?.link
+  const metaUrl =
+    buildData.data?.items.find((build: BuildItem) => !build.branch)?.link ||
```
Comment (arielj) [inline src/backend/storeManagers/gog/library.ts:681]:
```suggestion
    buildData.data?.items?.find((build: BuildItem) => !build.branch)?.link ||
```

do we need a `?` here? not sure, maybe we don't (I'm always confused what happens when chaining `?` operators)

Comment (imLinguin) [reply src/backend/storeManagers/gog/library.ts:681]: Maybe we should, I'm not sure how the response looks like when there are no builds available (that's the edge case but it doesn't hurt I guess.

**Pull request merged**